cmake/modules/FindGBM.cmake

```diff
@@ -29,11 +29,24 @@ find_package_handle_standard_args(GBM
                                   REQUIRED_VARS GBM_LIBRARY GBM_INCLUDE_DIR
                                   VERSION_VAR GBM_VERSION)
 
+include(CheckCSourceCompiles)
+set(CMAKE_REQUIRED_LIBRARIES ${GBM_LIBRARY})
+check_c_source_compiles("#include <gbm.h>
+
+                         int main()
+                         {
+                           gbm_bo_map(NULL, 0, 0, 0, 0, GBM_BO_TRANSFER_WRITE, NULL, NULL);
+                         }
+                         " GBM_HAS_BO_MAP)
```

Comment:
This is in mesa since version 12.0.0-rc1, so IMO you should just assume it is present.

Reply:
I wish that was possible but as said in the PR overview some versions of libmali don't support it